app.py

```diff
@@ -1,34 +1,37 @@
 from time import sleep
 
-from blessed import Terminal
+from rich.layout import Layout
 from rich.live import Live
 from rich.panel import Panel
 
 from src.resources.character import Character
+from src.resources.items import AbstractDungeonItem
 from src.resources.level import Level
 
-# Used to get player input
-term = Terminal()
 
-
-def run_game(panel: Panel) -> Panel:
+def run_game(layout: Layout) -> Panel:
     """
     This function in in charge of running the game. It will call update and draw for each game object.
 
-    Layout: Layout  Holds all the rich rederables for the game. Updated with a new panel each tick.
+    Layout: Layout  Holds all the rich renderables for the game. Updated with a new panel each tick.
     """
-    player.draw(level.board)
+    dungeon_item.draw(level.board)
+    player.draw()
     panel = Panel(level.to_string())
+    layout.update(panel)
     sleep(0.1)
-    return panel
 
 
 # This a temporary home for these game objects. they should be moved to a better place.
-level = Level(11, 10, [1, 2, 3, 4], [])
-player = Character(2, 2, "$")
+level = Level(10, 10, [1, 2, 3, 4], [])
+player = Character(level, "$")
+player.start()
+
+dungeon_item = AbstractDungeonItem(x=2, y=2, symbol="@", color="green")
```

Comment:
we should not be creating an instance of an abstract class. create an instance of a concrete derived class.

Reply:
Removed that code; fixed